tests/src/test/RegisterAddress.spec.ts

```diff
@@ -0,0 +1,55 @@
+// [object Object]
+// SPDX-License-Identifier: Apache-2.0
+
+import { ApiPromise, Keyring, WsProvider } from '@polkadot/api';
+
+describe('RegisterAddress', (): void => {
+  let api: ApiPromise;
+  let alice;
+
+  beforeEach(async () => {
+    process.env.NODE_ENV = 'test';
+
+    const provider = new WsProvider('ws://127.0.0.1:9944');
+
+    api = await ApiPromise.create({ provider });
+
+    const keyring = new Keyring({
+      type: 'sr25519'
+    });
+    const alice = keyring.addFromUri('//Alice');
+  });
+
+  afterEach(async () => {
+    await api.disconnect();
+  });
+
+  it('should work', async (): void => {
+    const unsub = await api.tx.creditcoin.registerAddress('Luniverse', '0xafafafafafafafafafafaf')
+        .signAndSend(alice, ({
```

Comment:
@nathanwhit, @jfrank-summit  I am getting an error here:

```bash
$ yarn test
yarn run v1.22.17
$ polkadot-dev-run-test --forceExit --runInBand
$ polkadot-dev-run-test --forceExit --runInBand
  console.warn
    2022-03-18 14:33:27        API/INIT: RPC methods not decorated: creditcoin_eventsSubscribe, creditcoin_eventsUnsubscribe, creditcoin_getEvents

      at apply (node_modules/@polkadot/util/logger.cjs:81:22)
      at Object.warn (node_modules/@polkadot/util/logger.cjs:161:14)
      at ApiPromise._filterRpcMethods (node_modules/@polkadot/api/base/Decorate.cjs:311:9)
      at ApiPromise._filterRpc (node_modules/@polkadot/api/base/Decorate.cjs:274:10)
      at ApiPromise._metaFromChain (node_modules/@polkadot/api/base/Init.cjs:350:10)

 FAIL  src/test/RegisterAddress.spec.ts
  RegisterAddress
    ✕ should work (935 ms)

  ● RegisterAddress › should work

    TypeError: Cannot read properties of undefined (reading 'sign')

      27 |   it('should work', async (): void => {
      28 |     const unsub = await api.tx.creditcoin.registerAddress('Luniverse', '0xafafafafafafafafafafaf')
    > 29 |         .signAndSend(alice, ({
         |          ^
      30 |             status,
      31 |             events = []
      32 |         }) => {

      at isKeyringPair (node_modules/@polkadot/api/util/isKeyringPair.cjs:13:40)
      at Submittable.#observeSign (node_modules/@polkadot/api/submittable/createClass.cjs:176:48)
      at node_modules/@polkadot/api/submittable/createClass.cjs:165:52
      at node_modules/@polkadot/api/promise/decorateMethod.cjs:67:26
      at decorateSubscribe (node_modules/@polkadot/api/promise/decorateMethod.cjs:63:10)
      at node_modules/@polkadot/api/promise/decorateMethod.cjs:86:23
      at Submittable.signAndSend (node_modules/@polkadot/api/submittable/createClass.cjs:166:8)
      at Object.<anonymous> (src/test/RegisterAddress.spec.ts:29:10)

Test Suites: 1 failed, 1 total
Tests:       1 failed, 1 total
Snapshots:   0 total
Time:        4.104 s, estimated 5 s
Ran all test suites.
Force exiting Jest: Have you considered using `--detectOpenHandles` to detect async operations that kept running after all tests finished?
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Am I reading this correctly that `.registerAddress` returns undefined ?  Do I need to configure something else in order for this to work ?

Reply:
@jfrank-summit you have the same snippet here https://github.com/gluwa/creditcoin-substrate/pull/86/files#diff-a1966c880054de29f3b8fb63f4169c22187219559a8f874426c73cbacd032ac3R140. How did you get that to work ?

Reply:
FTR I originally though that this had something to do with #89 but I don't think so anymore.